src/js/rerender.js

```diff
@@ -26,4 +27,26 @@ function rerender() {
     });
 }
 
-export { rerender };
+function rerenderExercises(params) {
+  serviceGetExercises(params)
+    .then(data => {
+      renderExercises(data.results);
+      renderPagination(data.totalPages);
+
+      Array.from(refs.pagination.childNodes).forEach(p => {
+        p.classList.remove('active');
+      });
+
+      const activePage = Array.from(refs.pagination.childNodes).find(
+        p => p.dataset.id == data.page
+      );
+      if (activePage) {
+        activePage.classList.add('active');
+      }
+    })
+    .catch(error => {
+      console.log(error);
```

Comment:
In the list of requirements there is a point: no console logs. Or you need to show some alert/notification, or just ignore error

Reply:
replaced with izitoast